Source/WTF/wtf/text/icu/UTextProviderLatin1.cpp

```diff
@@ -144,7 +144,7 @@ static UBool uTextLatin1Access(UText* uText, int64_t index, UBool forward)
     uText->chunkLength = static_cast<int32_t>(uText->chunkNativeLimit - uText->chunkNativeStart);
 
 WTF_ALLOW_UNSAFE_BUFFER_USAGE_BEGIN
-    StringImpl::copyCharacters(const_cast<UChar*>(uText->chunkContents), unsafeMakeSpan(static_cast<const LChar*>(uText->context) + uText->chunkNativeStart, uText->chunkLength));
+    StringImpl::copyCharacters(unsafeMakeSpan(const_cast<UChar*>(uText->chunkContents), uText->chunkLength), unsafeMakeSpan(static_cast<const LChar*>(uText->context) + uText->chunkNativeStart, uText->chunkLength));
```

Comment:
These kinds of getters that turn, say, `UText` data members into a span seem like they should be separate functions. They are the "boundary functions" that translate the unsafe API into our safer C++ idioms and I think they should contain as little else as possible.